crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -96,8 +98,8 @@ impl<I: SparseSetIndex, V> SparseArray<I, V> {
 pub struct ComponentSparseSet {
```

Comment:
I think that having some comments around here about why we don't store the entity generation would be nice, though I don't think it should block this PR since this was effectively already a problem :sweat_smile:

Reply:
Added!